### ISSUE TITLE
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.

### ISSUE THREAD

**Issue by faris-sketch** [nicknochnack/TFODCourse#121]
Hi, im getting this issue when installing tensorflow in Jupyter Notebook.
![image](https://user-images.githubusercontent.com/122259758/211356315-1d2e5c56-1d32-47b0-9169-c288de07e367.png)

**Comment by sabbir2609** [comment]
you wanna upgrade tensorflow ?

**Comment by faris-sketch** [comment]
> 
how do i upgrade tensorflow?
btw i used tensorflow 2

**Comment by sabbir2609** [comment]
i guess the best way is to create another virtual environment and install everything.
and python compatibility is important.

**Comment by abdulSamad57** [comment]
@faris-sketch @sabbir2609  did  you solve the issue 
i am getting the same errors could you guide me
i am using python 3.10
and tensor 2.10

**Comment by sabbir2609** [comment]
don't upgrade, Tensorflow dropped support for Windows from v2.11

**Comment by abdulSamad57** [comment]
yeah i am not upgrading but still getting alot of conflict  error should i downgrade the version of python

**Comment by sabbir2609** [comment]
try 3.9.6
